### ISSUE TITLE
to find work you love, don't (always) follow your passion

### ISSUE THREAD

**Issue by nelsonic** [dwyl/start-here#107]
![chose-a-job-you-love-never-work](https://cloud.githubusercontent.com/assets/194400/20655638/59c53f2e-b51c-11e6-81e7-2a261049883e.jpg)

Watch: https://youtu.be/MKlx1DLa9EA 
and read: https://80000hours.org/articles/dont-follow-your-passion
then _share_ your `#Thoughts` ...

**Comment by markwilliamfirth** [comment]
omg I was watching this last night *snap*

**Comment by nelsonic** [comment]
@markwilliamfirth yeah, your fellow Ox-Grads are doing a _decent_ job of helping people DWYL. ❤️ 
We are going to take it to a _whole_ other level by building **tools** that help people ***understand*** (_or **discover**_) what they love doing (_and whether they can create `value` by doing it_), ***connect*** with others who share the passion (_i.e. form a community_) and track progress towards ***mastery*** in the ***skills*** required to DWYL... 😉

**Comment by nelsonic** [comment]
[![image](https://user-images.githubusercontent.com/194400/27969931-d2ac8ac4-6344-11e7-8283-e943cc38d05f.png)](
http://ecorner.stanford.edu/videos/2105/Do-What-You-Love)

http://ecorner.stanford.edu/videos/2105/Do-What-You-Love

**Comment by nelsonic** [comment]
![image](https://user-images.githubusercontent.com/194400/27970125-8aa88b28-6345-11e7-81fc-e0117582c77a.png)
http://www.defencejobs.gov.au/women-in-adf/#/
😕

**Comment by chisNaN** [comment]
The idea of not following our passion is also explained [here](https://www.youtube.com/watch?v=DOS7bB050Bs&t=385s) in a simpler way.
Thanks Nelson for this dialectic approach)))

**Comment by nelsonic** [comment]
https://youtu.be/CJN-D6kdvws & https://youtu.be/m7Btde7e7Hs

**Comment by newswim** [comment]
https://www.theatlantic.com/business/archive/2015/08/do-what-you-love-work-myth-culture/399599

**Comment by nelsonic** [comment]
@newswim thanks for _reminding_ us of that article. 👍 
I've _ordered_ a copy of the book and will read it _voraciously_ when it arrives.

![image](https://user-images.githubusercontent.com/194400/27988425-501390fa-6419-11e7-84f4-b982a52dde4a.png)

**Comment by YvesMuyaBenda** [comment]
@nelsonic Personally, I would less take to task the individual and her publishing company, but I very much agree with the concept of a postive goal that "pulls one through" to something better.

**Comment by newswim** [comment]
@nelsonic I hope I didn't push you over the cliff of buying this apparently self-indulgent do-over thesis from someone, who after reading your comments and some of the other reviewers' comments, seems to have lost their own moral center.

It's misguided to vilify a whole enterprise (or worse, _[C]apitalism_) while at the same time failing to articulate the actual **flaws**, or offer alternatives. I can sympathize with a [classical liberal view](https://en.wikipedia.org/wiki/John_Stuart_Mill)-- pursue one's own interests and basically _do no harm_. But when scaled out, is this position causing harm? There's this whole concept of _Neo-Liberalism_ but I won't weight into that. Ironically, Tokumitsu's book almost sounds like an apologetics for the _Bourgeoisie_ (my own fumbling understanding of marxism), self-interested actors following her own advice. However they might be violating that second command, _do no harm_. Maybe there is tension between these two `commands`, and out of that a dissertation might've seemed plausible (speculating), which nonetheless failed due to these flaws in the criticism. Beyond that, just further speculation...

**Frankl** is a great recommendation, I have a copy and need to sit down with that for a few days (slow reader). I'll just add that I don't know much about Tokumitsu, and don't plan to investigate after reading that interview. But, I understand what it's like to make a drastic career change, and spend a decade dawdling in liberal arts before finding meaning in something more _hard_. We tend to undersell the notion of _self-work_ in exchange for "self-improvement", particularly in the US. Liberal education is important when it comes to gaining an understanding of human achievements and motivations.. but I think can be taught in a way which doesn't bring out the underlying urges, it's too "soft" in a way.

I want my hard sciences made more approachable and integrative, and my soft arts made more rigorous and formal. Damnit, I'm a centrist.

Anyway, please ignore me and just read [Maria Popova](https://www.brainpickings.org/)'s blog and _do as thou wilt_.

**Comment by YvesMuyaBenda** [comment]
@newsmim On the relationship between soft and hard, my basic thought is that programming/computer science is the most perfect playground for learning how to _build_ things. Insights I have gotten from the study of the topic has been applied to thoughts related to dance, yoga, poetry, grammar, music, and so on. In fact, building a computational model of a domain might be said to be the ultimate test of understanding, because you have to be super clear and explicit about how you are thinking about a topic.
